playbooks/common/openshift-master/config.yml

```diff
@@ -404,23 +385,3 @@
   tasks:
   - file: name={{ g_master_mktemp.stdout }} state=absent
     changed_when: False
-
```

Comment:
Looks like 'Enable cockpit' and 'Configure flannel' should also be in additional_config.yaml as well.